pulpcore/cli/container/repository.py

```diff
@@ -122,3 +133,46 @@ def sync(
         href=repository_href,
         body=body,
     )
+
+
+@repository.command(name="tag")
+@name_option
+@href_option
+@click.option("--tag", help=_("Name to tag an image with"), required=True, callback=_tag_callback)
+@click.option("--digest", help=_("SHA256 digest of the Manifest file"), required=True)
+@pass_repository_context
+def add_tag(
+    repository_ctx: PulpRepositoryContext,
+    digest: str,
+    tag: str,
+) -> None:
+    if not repository_ctx.capable("tag"):
+        raise click.ClickException(_("pulp_container 2.3.0 is required to tag images"))
```

Comment:
(Note for later:) Ha, we should have a `needs_capability` that raises itself.
Rationale: All mentions of Pulp versions should be in `context.py`.